mappings/net/minecraft/nbt/NbtHelper.mapping

```diff
@@ -10,27 +11,65 @@ CLASS net/minecraft/class_2512 net/minecraft/nbt/NbtHelper
 	FIELD field_33227 COMMA Ljava/lang/String;
 	FIELD field_33228 COLON C
 	METHOD method_10681 toBlockState (Lnet/minecraft/class_2487;)Lnet/minecraft/class_2680;
-		ARG 0 compound
+		COMMENT {@return the block state from the {@code nbt}}
+		COMMENT
+		COMMENT <p>This returns the default state for {@link net.minecraft.block.Blocks#AIR}
+		COMMENT if the block name is not present.
+		COMMENT
+		COMMENT @see #fromBlockState(BlockState)
+		ARG 0 nbt
 	METHOD method_10682 withProperty (Lnet/minecraft/class_2688;Lnet/minecraft/class_2769;Ljava/lang/String;Lnet/minecraft/class_2487;Lnet/minecraft/class_2487;)Lnet/minecraft/class_2688;
 		ARG 0 state
 		ARG 1 property
 		ARG 2 key
 		ARG 3 properties
 		ARG 4 root
 	METHOD method_10683 toGameProfile (Lnet/minecraft/class_2487;)Lcom/mojang/authlib/GameProfile;
-		ARG 0 compound
+		COMMENT {@return the game profile converted from {@code nbt}}
+		COMMENT
+		COMMENT @see #writeGameProfile(NbtCompound, GameProfile)
+		ARG 0 nbt
 	METHOD method_10684 writeGameProfile (Lnet/minecraft/class_2487;Lcom/mojang/authlib/GameProfile;)Lnet/minecraft/class_2487;
-		ARG 0 compound
+		COMMENT Writes the game profile to {@code nbt}. This modifies the passed compound.
+		COMMENT
+		COMMENT @return the compound with the serialized game profile
+		COMMENT @see #toGameProfile(NbtCompound)
+		ARG 0 nbt
 		ARG 1 profile
 	METHOD method_10685 nameValue (Lnet/minecraft/class_2769;Ljava/lang/Comparable;)Ljava/lang/String;
 		ARG 0 property
 		ARG 1 value
 	METHOD method_10686 fromBlockState (Lnet/minecraft/class_2680;)Lnet/minecraft/class_2487;
+		COMMENT {@return the serialized block state}
+		COMMENT
+		COMMENT @see #toBlockState(NbtCompound)
 		ARG 0 state
 	METHOD method_10687 matches (Lnet/minecraft/class_2520;Lnet/minecraft/class_2520;Z)Z
+		COMMENT {@return whether {@code subject} "matches" {@code standard}}
```

Comment:
You can just write `whether {@code standard} is a subset of of {@code subject}`

Reply:
```suggestion
		COMMENT {@return whether {@code standard} is a subset of {@code subject}}
```